spec/function.dd

```diff
@@ -2054,14 +2054,12 @@ void func(T)(T x)
 {
     writeln(x);
 }
-
```

Comment:
This is troubling. I don't think we should be deleting empty lines from any spec examples because of Latex bugs. Can you put some dummy whitespace here?